aspnet/web-api/samples-list.md

```diff
@@ -1,10 +1,11 @@
 ---
 uid: web-api/samples-list
-title: "Web API Samples List | Microsoft Docs"
+title: "Web API Samples List - ASP.NET 4.x"
 author: rick-anderson
-description: ""
+description: "ASP.NET Web API samples list for ASP.NET 4.x"
```

Comment:
Quotation marks aren't necessary unless you have a special character like a colon in the text.